modules/ROOT/pages/bat-install-task.adoc

```diff
@@ -5,10 +5,28 @@ endif::[]
 
 The Blackbox Automated Testing (BAT) CLI wrapper is a command line wrapper that you install to execute API tests.
 
-== Prerequisite
+== Prerequisite for Linux and MacOS Systems
 
 Before installing the BAT CLI, ensure that AdoptOpenJDK's OpenJDK 8 is installed on your system.
 
+== Prerequisites for Windows 10 Systems
+
+. Install Windows Subsystem Linux (WSL)
++
+The instructions are at https://docs.microsoft.com/en-us/windows/wsl/install-win10.
```

Comment:
For the sake of information maintenance, I wonder if we should not say only...:
```suggestion
Installation instructions are on the Microsoft website.
```
OR
```suggestion
Installation instructions are available at https://docs.microsoft.com.
```